generate_certs.py

```diff
@@ -0,0 +1,53 @@
+'''
+Check for all three certificates or create an account device along 
+with its certificates.
+'''
+
+import http_requests
+import argparse
+from os import makedirs, path
+
+devCert_URL = 'https://api.nrfcloud.com/v1/account/certificates'
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(description="Device Credentials Installer",
+                                    formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+    parser.add_argument("-p", "--path", type=str,
+                        help="Path to save files.  Selects -s", default="./")
+    return parser.parse_args()
+
+args = parse_args()
+
+def write_file(pathname, filename, string):
+    '''save string to file'''
+    if not path.isdir(pathname):
+        try:
+            makedirs(pathname, exist_ok=True)
+        except OSError as e:
+            raise RuntimeError("Error creating file path", e)
+    full_path = path.join(pathname, filename)
+    try:
+        f = open(full_path, "w")
+    except OSError:
+        raise RuntimeError("Error opening file: " + full_path)
+    f.write(string)
+    print("File created: " + path.abspath(f.name))
+    f.close()
+    return
+
+def create_device(account_type, api_key, client_cert, priv_key):
+   
+    create_dev_cert = http_requests.http_req('POST', devCert_URL, api_key)  #create uses 'POST'
```

Comment:
![http_error_msg_window](https://user-images.githubusercontent.com/107961089/191130726-2e66220f-bec0-43f5-a6c9-1a315c9380bc.png)
Added to program so that if we ran through any of the "except" functions in http_requests the error window will pop up.